wcivf/apps/elections/models.py

```diff
@@ -239,6 +243,19 @@ def pluralized_division_name(self):
 
         return pluralise.get(suffix, f"{suffix}s")
 
+    @property
+    def has_results(self):
+        """
+        Returns a boolean for if the election has results
+        """
+        return bool(
+            self.spoilt_ballots
+            or self.ballot_papers_issued
+            or self.turnout
+            or self.electorate
```

Comment:
I think this is what you suggested? @symroe

Reply:
I think the last clause can also be a n `or`?

Reply:
I'm not sure about that; I didn't think we wanted to display the table unless at least one of the above fields are not `None`.

Reply:
I've renamed this for clarity `has_election_results`.